recipes-multimedia/gstreamer/gstreamer1.0-plugins-bad_1.16.imx.bb

```diff
@@ -63,6 +66,7 @@ PACKAGECONFIG[hls]             = "-Dhls=enabled -Dhls-crypto=nettle,-Dhls=disabl
 # to -base yet. They depend on the gstgl library in -base, so we do
 # not add GL dependencies here, since these are taken care of in -base.
 PACKAGECONFIG[gl]              = "-Dgl=enabled,-Dgl=disabled,"
+PACKAGECONFIG[gles2]           = ",,virtual/libgles2"
```

Comment:
Doesn't this use any build option for it? auto detection is bad so if we can "force" it, it is always nicer.

Reply:
Hello @otavio, thanks for your reply. I didn't find any build option for this in gstreamer (so i took the same options in gstreamer1.0-plugins-base).

I propose this as commit log, it is good ? : 

gstreamer1.0-plugins-bad: Let processors with pxp chose the correct Opengl API

The processors without gpu (imx6ul/imx6ull) support only opengl-es.The system must be able to chose the correct API instead of forcing it for all processors.

Signed-off-by: ZIDOUH Zakaria <zakaria.zidouh@gmail.com>

Reply:
Yes, the commit log makes sense. Please pay attention to the number of columns so it fits on the good practices.

@thochstein please review this commit as well.